roles/controller_casc_from_aap/templates/current_projects_improved.j2

```diff
@@ -0,0 +1,20 @@
+---
+controller_projects:
+{% for project in current_pj_asset_value %}
+  - name: "{{ project.name }}"
+    description: "{{ project.description }}"
+    organization: "{{ project.organization.name | default('ToDo: The project \'' + project.name + '\' must have an organization') }}"
+{% if project.scm_type %}
+    scm_type: "{{ project.scm_type }}"
+    scm_url: "{{ project.scm_url }}"
+    scm_credential: "{{ project.credential.name | default('ToDo: The project \'' + project.name + '\' is missing the SCM credential') }}"
```

Comment:
scm_credential: "{{ project.summary_fields.credential.name | default('ToDo: The project \'' + project.name + '\' is missing the SCM credential') }}"

Reply:
Updated